<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HMCChannelWidget</class>
 <widget class="QWidget" name="HMCChannelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbChannel">
     <property name="title">
      <string>Channel Title</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,2,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,10">
        <item>
         <widget class="QLabel" name="lbVoltage">
          <property name="text">
           <string>Voltage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdVoltage">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,10">
        <item>
         <widget class="QLabel" name="lbCurrent">
          <property name="text">
           <string>Current</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdCurrent">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbOutEnable">
        <property name="text">
         <string>Output Enable</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnSetVoltage">
          <property name="text">
           <string>Set Voltage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSetCurrent">
          <property name="text">
           <string>Set Current</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
